CMakeLists.txt

```diff
@@ -1217,6 +1217,29 @@ else ()
   set(HAVE_HALF_FLOAT OFF)
 endif ()
 
+# Define a macro to check if we have included and initialized libipt.
+set(HAVE_LIBIPT OFF)
+# Initialize libipt and activate the macro HAVE_LIBIPT.
+find_package(Git QUIET)
+if (GIT_FOUND AND EXISTS "${PROJECT_SOURCE_DIR}/.git")
+  # Update submodules as needed
+  option(UPDATE_SUBMODULE "Update submodules during build" ON)
+  if (UPDATE_SUBMODULE)
+    message(STATUS "Update submodules")
+    execute_process(COMMAND ${GIT_EXECUTABLE} submodule update --init --recursive
```

Comment:
I think I had the same comment earlier: I don't think the build process should be doing this.  The user should be responsible for initial setup and updating, and should be able to build with any desired version including old versions.

Reply:
Done.